.github/workflows/on-pr.yml

```diff
@@ -144,23 +144,7 @@ jobs:
         if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
         run: exit 0
       - name: CI failed
-        if: ${{ needs.ci.result != 'success' }}
+        if: ${{ github.event.pull_request.head.repo.full_name == github.repository && needs.ci.result != 'success' }}
```

Comment:
Can't we just have this condition and not need the Community PR step at all? Either this step runs first and errors, or its skipped and we get CI succeed step.

Reply:
Yep -- I thought this might be clearer but realised it only affects the shell output, not the list of checks in the UI, so have collapsed them.